examples/basic/outputs.tf

```diff
@@ -3,28 +3,38 @@
 ########################################################################################################################
 
 output "account_id" {
-  description = "Account ID of the watsonx Governance instance"
+  description = "Account ID of the watsonx.governance instance"
   value       = module.watsonx_governance.account_id
 }
 
 output "crn" {
-  description = "CRN of the watsonx Governance instance"
+  description = "CRN of the watsonx.governance instance"
   value       = module.watsonx_governance.crn
 }
 output "guid" {
-  description = "GUID of the watsonx Governance instance"
+  description = "GUID of the watsonx.governance instance"
   value       = module.watsonx_governance.guid
 }
 output "name" {
-  description = "Name of the watsonx Governance instance"
+  description = "Name of the watsonx.governance instance"
   value       = module.watsonx_governance.name
 }
 
+output "plan_id" {
+  description = "Plan ID of the watsonx Orchestrate instance."
+  value       = module.watsonx_governance.plan_id
+}
+
+output "dashboard_url" {
+  description = "Dashboard URL of the watsonx Orchestrate instance."
```

Comment:
should be governance not orchestrate

Reply:
Thank you for pointing this oversight. I have corrected.